templates/ui-service.yaml

```diff
@@ -23,7 +18,10 @@ spec:
     app.kubernetes.io/name: {{ include "vault.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     component: server
-  publishNotReadyAddresses: true
+    {{- if .Values.ui.activeVaultPodOnly }}
```

Comment:
This is only true if in HA mode. I think we don't even need a configurable here.  If it's HA mode, just set the selector criteria, else point to the single deployed pod.

Thoughts?

Reply:
Sure, I will make the change.

Reply:
I can add an additional condition to check that HA mode is on.

I think this should be configurable to handle other possible use cases like https://github.com/hashicorp/vault-helm/issues/326. What do you think? I could remove this option if you think otherwise.

Reply:
I think configurable is fine.

Reply:
I've added the check for HA in 956b101